### PR TITLE
[14.0][IMP] account_invoice_report_due_list: added outstanding amounts

### PR DESCRIPTION
**Pull request by dalonsod**
With this improvement due list also shows outstanding amount for the existing dues:

e.g. for an already paid invoice:

imagen

This PR implements #234

### CODE REVIEW

**Comment by dalonsod** [comment]
Hello @victoralmau changes suggested in #234 made here.

Codecov is not passed, I don't know why (added extra test code for it, but it didn't seem to work), could you help me?

cc @carlosdauden

**Comment by dalonsod** [comment]
Don't worry, finally I've found the problem and solved it.

Could you review? Thanks!!!

File: account_invoice_report_due_list/tests/test_invoice_report_due_list.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def setUpClass(cls):
                 "reconcile": True,
             }
         )
+        cls.eur = cls.env.ref("base.EUR")
```
Comment (victoralmau) [inline account_invoice_report_due_list/tests/test_invoice_report_due_list.py]:
```suggestion
        usd = cls.env.ref("base.USD")
        eur = cls.env.ref("base.EUR")
        cls.currency = cls.env.ref("base.main_company").currency_id
        cls.currency_extra = eur if cls.currency == usd else usd
```

File: account_invoice_report_due_list/tests/test_invoice_report_due_list.py [hunk 2]
```diff
@@ -95,3 +96,16 @@ def test_due_list(self, move_type="out_invoice"):
...
+        )
+        move_form2.partner_id = self.partner
+        move_form2.invoice_payment_term_id = self.payment_term_multi
+        move_form2.currency_id = self.eur
+        with move_form2.invoice_line_ids.new() as line_form2:
+            line_form2.product_id = self.product_id
+            line_form2.price_unit = 200.0
+        invoice2 = move_form2.save()
+        invoice2.action_post()
+        self.assertEqual(invoice2.get_multi_due_list()[0][2], 50.0)
```
Comment (victoralmau) [inline account_invoice_report_due_list/tests/test_invoice_report_due_list.py]:
```suggestion
        move_form2 = Form(
            self.env["account.move"].with_context(default_move_type=move_type)
        )
        move_form2.partner_id = self.partner
        move_form2.invoice_payment_term_id = self.payment_term_multi
        with move_form2.invoice_line_ids.new() as line_form2:
            line_form2.product_id = self.product_id
            line_form2.price_unit = 200.0
        invoice2 = move_form2.save()
        invoice2.action_post()
        self.assertEqual(invoice2.get_multi_due_list()[0][2], 200.0)
        
   def test_due_list_currency_extra(self):
           move_form2 = Form(
            self.env["account.move"].with_context(default_move_type=move_type)
        )
        move_form2.partner_id = self.partner
        move_form2.invoice_payment_term_id = self.payment_term_multi
        move_form2.currency_id = self.currency_extra
        with move_form2.invoice_line_ids.new() as line_form2:
            line_form2.product_id = self.product_id
            line_form2.price_unit = 200.0
        invoice2 = move_form2.save()
        invoice2.action_post()
        self.assertEqual(invoice2.get_multi_due_list()[0][2], 50.0)
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Thanks for your contribution.

**Comment by dalonsod** [comment]
Comments attended!

**Review by carlosdauden — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge major

Please fw-port it to v15

**Comment by dalonsod** [comment]
> Please fw-port it to v15

#238 ready for reviews

**Pull request merged**